Components/navbar/navbar.css

```diff
@@ -23,17 +24,56 @@
 }
 
 .hamburger {
+  display: block;
   cursor: pointer;
   font-size: 2rem;
 }
 
+.nav-links {
+  position: absolute;
+  left: 0;
+  right: 0;
+  /* top: -180px; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 8rem;
+  background-color: var(--primary);
+  z-index: 1;
+  transition: 0.3s all;
+}
+
+.nav-links a {
+  margin: 0 1rem;
+}
+
+.nav-links-not-visible {
+  top: -180px;
+}
+
+.nav-links-visible {
+  top: 5rem;
+}
+
 @media only screen and (min-width: 720px) {
   .big-logo {
     display: block;
     height: 5rem;
     margin-left: 2rem;
   }
 
+  #hamburger {
+    display: none;
+  }
+
```

Comment:
avoid using id selectors. It will cause problems in the future.

Reply:
Done